packages/frontend-v2/pages/home.tsx

```diff
@@ -65,13 +77,14 @@ const HomePage: NextPage = () => {
     logger.error("HomePage", error);
     handleApiClientError(error, router);
 
-    // render just the boiler plate page since we couldn't fetch the student w plans
-    return <PageLayout />;
+    // If we couldn't fetch the student's plan, show a blank page for now.
+    // We might want to show some more actionable error in the future.
+    return <div></div>;
```

Comment:
Is there a reason why you guys got rid of the `<PageLayout />`?

Reply:
having page layout here would render the sidebar rather than an empty page. i don't think either is necessarily incorrect, just not sure yet how we want to display to a user that their plan isn't loaded. i don't think it makes a huge difference either way at the moment tho